homeassistant/components/person/__init__.py

```diff
@@ -0,0 +1,168 @@
+"""
+Support for tracking people.
+
+For more details about this component, please refer to the documentation.
+https://home-assistant.io/components/person/
+"""
+import logging
+
+import voluptuous as vol
+
+from homeassistant.components.device_tracker import (
+    DOMAIN as DEVICE_TRACKER_DOMAIN)
+from homeassistant.const import ATTR_LATITUDE, ATTR_LONGITUDE
+from homeassistant.core import callback
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.entity_component import EntityComponent
+from homeassistant.helpers.event import async_track_state_change
+from homeassistant.helpers.restore_state import RestoreEntity
+
+_LOGGER = logging.getLogger(__name__)
+ATTR_FIRST_NAME = 'first_name'
+ATTR_LAST_NAME = 'last_name'
```

Comment:
Do we want first and last name? Or should we just stick with name?

Reply:
Yeah, name is probably enough.